blivet/util.py

```diff
@@ -637,3 +643,105 @@ def eintr_retry_call(func, *args):
             if e.errno == errno.EINTR:
                 continue
             raise
+
+#
+# Deprecation decorator.
+#
+_DEPRECATION_MESSAGE = "will be removed in a future version."
+def _default_deprecation_msg(func):
+    return "%s %s" % (func.__name__, _DEPRECATION_MESSAGE)
+
+def indent(text, spaces=4):
```

Comment:
Looks like I'll have to use the TextWrapper class if I want it to be portable to python2. I'll look into that.